exasol/shared_memory_vault.py

```diff
@@ -0,0 +1,226 @@
+from typing import Optional
+from multiprocessing.shared_memory import SharedMemory
+from math import ceil
+from datetime import datetime
+import struct
+
+
+default_key = '1001011001100101'
```

Comment:
```suggestion
polynomial = '1001011001100101'
```

Reply:
'key' was a bad name. Renamed to 'crc_divisor'.